src/KintoID.sol

```diff
@@ -484,21 +491,22 @@ contract KintoID is
 
     /**
      * @dev Hook that is called before any token transfer. Allow only mints and burns, no transfers.
-     * @param from source address
      * @param to target address
-     * @param batchSize The first id
+     * @param firstTokenId The first id
      */
-    function _beforeTokenTransfer(address from, address to, uint256 firstTokenId, uint256 batchSize)
+    function _update(address to, uint256 firstTokenId, address auth)
         internal
         virtual
         override(ERC721Upgradeable, ERC721EnumerableUpgradeable)
+        returns (address)
     {
+        address from = _ownerOf(firstTokenId);
```

Comment:
and it does not receive the `from` anymore but we can get it like this, double check please!

Reply:
this makes sense